helper/common/common.go

```diff
@@ -174,3 +174,14 @@ func PadLeftOrTrim(bb []byte, size int) []byte {
 
 	return tmp
 }
+
+// ExtendByteSlice extends given byte slice by needLength parameter and trims it
+func ExtendByteSlice(b []byte, needLength int) []byte {
```

Comment:
Could you add a couple of unit tests for the function?

Reply:
:heavy_check_mark: https://github.com/0xPolygon/polygon-edge/pull/1058/commits/b776177b72378fdc6a2497a769b4f6c4fafd1058